DESCRIPTION

```diff
@@ -91,7 +91,8 @@ Imports:
     tinytex (>= 0.11),
     xfun (>= 0.15),
     methods,
-    stringr (>= 1.2.0)
+    stringr (>= 1.2.0),
+    vctrs (>= 0.3.2)
```

Comment:
I'm hesitant on introducing a new dependency only for the purpose of supporting a specific feature `html_document(df_print = "paged")`. I'm okay if it is in `Suggests`, and we call it conditionally, e.g., `if (xfun::loadable('vctrs')) return(vctrs:: vec_ptype_abbr(x))`.

If the difference between the output of `rmarkdown:::paged_table_type_sum()` and `vctrs::vec_ptype_abbr()` is minor, I tend to fix the difference to make them consistent.

Reply:
Thank you, I moved `vctrs` to suggests and included the check whether the package is installed. This should work well since anyone who wants to make use of vctrs in a paged table will have it installed anyways. Could also skip the suggests altogether since `tibble` is already on the suggests list for `rmarkdown` and `tibble` depends on `vctrs`. 

With this, these kind of `vctrs`-type vector implementations now print similarly in standard output and paged tables:

```{r}
library(vctrs)
percent <- function(x = double()) {
  vec_assert(x, double())
  new_vctr(x, class = "percent")
}
format.percent <- function(x, ...) {
  paste0(formatC(signif(vec_data(x) * 100, 3)), "%")
}
vec_ptype_abbr.percent <- function(x, ...) "prcnt"

df <- tibble::tibble(x = percent(runif(4)))
```
`df` prints as:

```
## # A tibble: 4 x 1
##         x
##   <prcnt>
## 1   14.1%
## 2   51.5%
## 3   65.5%
## 4   95.5%
```

and `rmarkdown::paged_table(df)` now also as 

![screenshot](https://user-images.githubusercontent.com/5498966/96324700-55b5c980-0fe0-11eb-8d27-459ce67afbc0.png)